pkgs/development/libraries/gecode/3.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "gecode-${version}";
-  version = "3.7.3";
+  version = "6.0.0";
```

Comment:
this package is exposed as `gecode_3`. I don't think we should upgrade this one.